### ISSUE TITLE
Searching for inspections failed: undefined method `map' for nil:NilClass

### ISSUE THREAD

**Issue by sushantDev** [CocoaPods/CocoaPods#12444]
* [ ] I've read and understood the [*CONTRIBUTING* guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

# Report

## What did you do?
pod install

## What did you expect to happen?
open app in simulator

## What happened instead?

Error output from CocoaPods:
↳
    Searching for inspections failed: undefined method `map' for nil:NilClass

Error running pod install

### Stack

```
   CocoaPods : 1.15.2
        Ruby : ruby 2.6.10p210 (2022-04-12 revision 67958) [universal.arm64e-darwin23]
    RubyGems : 3.0.3.1
        Host : macOS 14.5 (23F79)
       Xcode : 15.4 (15F31d)
         Git : git version 2.39.3 (Apple Git-146)
Ruby lib dir : /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib
Repositories : trunk - CDN - https://cdn.cocoapods.org/
```

### Installation Source

```
Executable Path: /usr/local/bin/pod
```

### Plugins

```
cocoapods-deintegrate : 1.0.5
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.1
cocoapods-trunk       : 1.6.0
cocoapods-try         : 1.2.0

**Comment by taimoor522** [comment]
I am facing th same issue: 

### Stack

    ```
       CocoaPods : 1.15.2
            Ruby : ruby 2.6.10p210 (2022-04-12 revision 67958) [universal.x86_64-darwin23]
        RubyGems : 3.0.3.1
            Host : macOS 14.5 (23F79)
           Xcode : 15.4 (15F31d)
             Git : git version 2.15.0
    Ruby lib dir : /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib
    Repositories : trunk - CDN - https://cdn.cocoapods.org/
    ```

    ### Plugins

    ```
    cocoapods-clean       : 0.0.1
    cocoapods-deintegrate : 1.0.5
    cocoapods-plugins     : 1.0.0
    cocoapods-search      : 1.0.1
    cocoapods-trunk       : 1.6.0
    cocoapods-try         : 1.2.0

**Comment by taimoor522** [comment]
@sushantDev I have solved it my upgrading my ruby version to 3.3.3 using Ruby Version Manager (RVM)

**Comment by sushantDev** [comment]
Already solved. Thanks (RVM) did the job for me too. 😇
